scripts/uploaded-files/record-existing-files.ts

```diff
@@ -8,78 +8,18 @@
 import '../../server/env';
 
 import { Command } from 'commander';
+import { pickBy } from 'lodash';
 import PQueue from 'p-queue';
 
 import { getFileInfoFromS3 } from '../../server/lib/awsS3';
 import logger from '../../server/lib/logger';
+import { FileFieldsDefinition } from '../../server/lib/uploaded-files';
 import { parseToBoolean } from '../../server/lib/utils';
 import models, { sequelize } from '../../server/models';
 import UploadedFile, { FileKind, SUPPORTED_FILE_TYPES } from '../../server/models/UploadedFile';
 
 const DRY_RUN = process.env.DRY_RUN ? parseToBoolean(process.env.DRY_RUN) : true;
 
-const SIMPLE_FIELDS: Array<{
```

Comment:
Editing this file to extract the fields definitions to re-use them in different places